docs/install-and-use.md

```diff
@@ -93,6 +93,26 @@ If your application does use Slimmer/Static:
 //= require govuk_publishing_components/components/button
 ```
 
+As of v5.0.0, the target for browser support for govuk-frontend has changed to browsers that support using the `type="module"` attribute on `script` tags. This means that that govuk_publishing_components Javascript files that include files govuk-frontend will now error ungracefully and potentially function in an unpredictable way if a browser that doesn't support `type="module"` tries to evaluate the Javascript in a script tag without the `type="module"` attribute.
+
+To ensure that browsers that are below the target don't evaluate the Javascript that they don't support, consider using a separate file called `es6-components.js` for these components:
+
+```ruby
+# es6-components.js
+//= require govuk_publishing_components/lib
+//= require govuk_publishing_components/components/accordion
```

Comment:
Is it worth including a list of components for which this is the case, or pointing to a list somewhere? (it's in the auditing tools, for example, where it'll be automatically updated).

Reply:
Yes good point, I've updated the docs to reference the auditing tools:

> **Note:** You can get an up-to-date list of components that use govuk-frontend JS using the [component auditing tools](./auditing.md), components that use JS from govuk-frontend will be marked as "Yes" under the "GF JS" column.